styles.css

```diff
@@ -1,798 +1,798 @@
-    line-height: 20px;
-  }
-
-  .special {
-    padding-top: 21%;
-    width: inherit;
-    border-top-right-radius: 55pt;
-  }
-
-  .list-nav a:hover,
-  .list-nav a:active {
-    border-bottom: 0.3rem solid #6070ff;
-  }
-
-  .connect-1 {
-    font-style: normal;
-    font-weight: 700;
-    font-size: 40px;
-    line-height: 52px;
-    color: #091e42;
-  }
-
-  .about-p {
-    font-style: normal;
-    font-weight: 500;
-    font-size: 16px;
-    line-height: 24px;
-    color: #7f8cff;
-  }
-
-  /* HEADLINE PART */
-  .container-all {
-    opacity: 3;
-    width: 100%;
-    border-radius: 0;
-    padding-top: 23%;
-    gap: 2.2rem;
-    padding-bottom: 12%;
-    border-bottom-left-radius: 75pt;
-  }
-
-  .iam {
-    margin-top: 10;
-    font-style: normal;
-    font-weight: 700;
-    font-size: 48px;
-    line-height: 60px;
-  }
-
-  .end-2 {
-    width: 100%;
-  }
-
-  /* PROJECT PART */
-
-  .card-works {
-    display: grid;
-    grid-template-columns: 1fr 1fr;
-    column-gap: 6%;
-    padding: 2.8rem;
-    gap: 6%;
-  }
-
-  .link-item:nth-child(3) {
-    display: inline-block;
-  }
-
-  .card-works:nth-child(even) > .first-img {
-    order: 2;
-  }
-
-  .btn {
-    justify-self: left;
-  }
-
-  /* ABOUT PART  */
-  .languages-inputs {
-    flex-direction: row;
-  }
-
-  .languages-inputs li {
-    list-style: none;
-    flex-direction: column;
-    align-items: flex-start;
-  }
-
-  /* ABOUT PART  */
-
-  .works1 {
-    padding: 5%;
-    margin: 0;
-    display: flex;
-    flex-direction: row;
-    row-gap: 14%;
-  }
-
-  .skill-1 {
-    margin-top: -2rem;
-  }
-
-  /* CONTACT PART */
-
-  .contact-desc {
-    margin: auto 25%;
-  }
-
-  .space {
-    gap: 3%;
-  }
-
-  .connect {
-    font-style: normal;
-    font-weight: 500;
-    font-size: 16px;
-    line-height: 24px;
-    color: #7f8cff;
-  }
-
-  .contact-input,
-  .contact-message {
-    width: 44.7rem;
-    border-radius: 0.2rem;
-  }
-}
+body {
+  text-transform: capitalize;
+  background-color: #e5e5e5;
+  box-sizing: border-box;
+  overflow: scroll;
+}
+
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+  list-style-type: none;
+}
+
+.container {
+  margin: auto 2.4rem;
+  overflow: hidden;
+}
+
+header {
+  background-color: #fff;
+}
+
+html {
+  font-size: 62.3%;
+  scroll-behavior: smooth;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 10vh;
+  padding: 5%;
+  top: 0;
+}
+
+.list-nav {
+  display: none;
+}
+
+.link {
+  margin-bottom: 3.4rem;
+}
+
+a {
+  text-decoration: none;
+}
+
+.logo a {
+  font-size: 1.2rem;
+  font-weight: 700;
+}
+
+.logo {
+  color: #6070ff;
+  font-size: 1.8rem;
+  font-weight: 700;
+  text-decoration: none;
+}
+
+.container-all {
+  display: flex;
+  flex-direction: column;
+  background-image: url('./images/Header\ bg@2x.png');

```

Comment:
- [ ] Please **As suggested by the previous reviewer, ☝️ ☝️**, Still the background image for the main container is not visible 😨, Kindly check the link to the background Image and fix the issue to replicate the Figma design as much as possible 🙏 

**Currently**
![image](https://github.com/hetrox8/portfolio1/assets/102757126/3de7db50-4060-4bfd-b2bb-200548a7c025)

**This is not the last review for this project, Kindly make this simple adjustment and submit again for an extra review 🙏, in this module we are allowed to use 3 extra reviews, please take your time and make the suggested changes 😎**